src/lib/mdns/tests/TestServicePoolDriver.cpp

```diff
@@ -0,0 +1,13 @@
+#include <nlunit-test.h>
```

Comment:
nit: after #3888 the Drivers can be generated automatically. Requires to move part of this code into the test file.

Probably better to standardize how tests look and have main set the output to CSV and call out the test function.